sqlite.sql

```diff
@@ -172,6 +172,7 @@ CREATE TABLE IF NOT EXISTS `remoteusers` (
   actor_id TEXT NOT NULL,
   inbox TEXT NOT NULL,
   shared_inbox TEXT NOT NULL,
+  handle TEXT DEFAULT '' NOT NULL,
```

Comment:
No need to add this to the schema files, as the migrations run on install and take care of this.

Reply:
Reverted to previous